.circleci/config.yml

```diff
@@ -96,6 +97,69 @@ job-references:
             npm ci
             npm run grunt build
             export WP_TESTS_DIR=<< pipeline.parameters.wp_core_develop_dir >>/tests/phpunit
+            if [[ "$INCLUDE_MU_TESTS" == "1" ]]; then
+              sed -i "s|bootstrap=\"tests/phpunit/includes/bootstrap.php\"|bootstrap=\"$HOME/project/tests/bootstrap.php\"|" phpunit.xml.dist
```

Comment:
Does replacing the core bootstrap file cause the later test failures? Our bootstrap file doesn't necessarily mean that all of mu-plugins has been loaded, just the tested stuff, so it's not a 1:1 representation of what actually runs in production.

What we really need to do is make sure the mu-plugin loading code from core's wp-settings.php is running during these tests. Maybe we can modify the wp-settings.php from the tests to include core's mu-plugin loading code?